conformance/reports/v1.3.0/projectcontour-contour/experimental-v1.33.0-default-report.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: gateway.networking.k8s.io/v1
+date: "2025-11-13T13:49:19-05:00"
+gatewayAPIChannel: experimental
+gatewayAPIVersion: v1.3.0
+implementation:
+  contact:
+  - '@projectcontour/maintainers'
+  organization: projectcontour
+  project: contour
+  url: https://projectcontour.io/
+  version: v1.33.0
+kind: ConformanceReport
+mode: default
+profiles:
+- core:
+    result: success
+    statistics:
+      Failed: 0
+      Passed: 12
+      Skipped: 0
+  extended:
+    result: success
+    statistics:
+      Failed: 0
+      Passed: 1
+      Skipped: 0
+    supportedFeatures:
+    - GatewayAddressEmpty
+    - GatewayHTTPListenerIsolation
+    - GatewayInfrastructurePropagation
+    - GatewayPort8080
+    unsupportedFeatures:
+    - GatewayStaticAddresses
+  name: GATEWAY-GRPC
+  summary: Core tests succeeded. Extended tests succeeded.
+- core:
+    result: partial
+    skippedTests:
+    - HTTPRouteHTTPSListener
```

Comment:
had to leave this as a skipped test since there is not a specific enough feature to disable this one test (otherwise we would have to disable testing HTTPRoute in general), this should be addressed in the next release however to remove the skip entirely